src/command/client/search/interactive.rs

```diff
@@ -429,6 +429,11 @@ pub async fn history(
         let initial_input = app.input.as_str().to_owned();
         let initial_filter_mode = app.filter_mode;
 
+        {
+            #[cfg(target_os = "windows")]
+            let _ = event::read();
+        };
+
```

Comment:
What's this for?

Reply:
It was reading double input and was reading the Enter input when you enter the command